### ISSUE TITLE
Info tab for groups

### ISSUE THREAD

**Issue by CLBuechler2324** [barberscore/barberscore-api#568]
Should the Info tab for groups have a Submit or Save button? I had group ask when it couldn’t update. I did it for them on backend in Groups. Obviously not a critical item.
<img width="920" alt="Screen Shot 2019-09-07 at 6 38 13 PM" src="https://user-images.githubusercontent.com/48444276/64480946-5d315f00-d1a0-11e9-8e29-4b7d54935f74.png">

**Comment by dbinetti** [comment]
no, not going to add a submit button.  it's not a form; it just patches to
the API.

works in my testing.  are you certain a quartet admin tried to do the
updating?  I logged in as "loren shevitz" and it world.

**Comment by CLBuechler2324** [comment]
Loren Shevitz is quartet admin and tried. I used Auth0 as Loren and also tried.  Glad it worked for you, but concerns me with inconsistent response.

**Comment by dbinetti** [comment]
Hm.  Ok I’ll look more closely.

**Comment by CLBuechler2324** [comment]
Okay, it is working.  I guess Loren and I were just having bad day. Sorry.
